### ISSUE TITLE
您好，我想用自己的数据集训练

### ISSUE THREAD

**Issue by Xiao-YiZi** [jindongwang/transferlearning#265]
您好，我想用自己的数据集训练DeepDA，但是所有方法跑出来结果一直在10%左右，参数也调整过，没有太大变化，请问可以说一下文件夹具体结构吗？
下面是运行的log,我打印了源域和目标域的loader长度和类别数都是正确的。
Namespace(backbone='resnet50', batch_size=16, config='DAAN/DAAN.yaml', data_dir='/home/yxy/transfer-learning/wangjindong/data/multi_disguise_data', device=device(type='cuda'), early_stop=0, epoch_based_training=False, lr=0.001, lr_decay=0.75, lr_gamma=0.0003, lr_scheduler=True, momentum=0.9, n_epoch=10, n_iter_per_epoch=300, num_workers=0, seed=2021, src_domain='normal', tgt_domain='fast', transfer_loss='daan', transfer_loss_weight=1.0, use_bottleneck=True, weight_decay=0.0005)
764  #len(source loader)
9     #class number
546   #len(target loader)
Epoch: [ 1/10], cls_loss: 2.2115, transfer_loss: 0.0538, total_Loss: 2.2652, test_loss 2.215668, test_acc: 9.6959
Epoch: [ 1/10], cls_loss: 2.2115, transfer_loss: 0.0538, total_Loss: 2.2652, test_loss 2.215668, test_acc: 9.6959
Epoch: [ 2/10], cls_loss: 2.2086, transfer_loss: 0.0543, total_Loss: 2.2629, test_loss 2.213673, test_acc: 9.9017
Epoch: [ 2/10], cls_loss: 2.2086, transfer_loss: 0.0543, total_Loss: 2.2629, test_loss 2.213673, test_acc: 9.9017
Epoch: [ 3/10], cls_loss: 2.2066, transfer_loss: 0.0543, total_Loss: 2.2609, test_loss 2.213293, test_acc: 9.8674
Epoch: [ 3/10], cls_loss: 2.2066, transfer_loss: 0.0543, total_Loss: 2.2609, test_loss 2.213293, test_acc: 9.8674
Epoch: [ 4/10], cls_loss: 2.2053, transfer_loss: 0.0543, total_Loss: 2.2595, test_loss 2.211732, test_acc: 9.8559
Epoch: [ 4/10], cls_loss: 2.2053, transfer_loss: 0.0543, total_Loss: 2.2595, test_loss 2.211732, test_acc: 9.8559
Epoch: [ 5/10], cls_loss: 2.2030, transfer_loss: 0.0543, total_Loss: 2.2573, test_loss 2.210998, test_acc: 10.0732
Epoch: [ 5/10], cls_loss: 2.2030, transfer_loss: 0.0543, total_Loss: 2.2573, test_loss 2.210998, test_acc: 10.0732
Epoch: [ 6/10], cls_loss: 2.2041, transfer_loss: 0.0543, total_Loss: 2.2583, test_loss 2.209324, test_acc: 9.9245
Epoch: [ 6/10], cls_loss: 2.2041, transfer_loss: 0.0543, total_Loss: 2.2583, test_loss 2.209324, test_acc: 9.9245
Epoch: [ 7/10], cls_loss: 2.2018, transfer_loss: 0.0543, total_Loss: 2.2560, test_loss 2.209061, test_acc: 10.1532
Epoch: [ 7/10], cls_loss: 2.2018, transfer_loss: 0.0543, total_Loss: 2.2560, test_loss 2.209061, test_acc: 10.1532
Epoch: [ 8/10], cls_loss: 2.2011, transfer_loss: 0.0542, total_Loss: 2.2554, test_loss 2.207802, test_acc: 10.4505
Epoch: [ 8/10], cls_loss: 2.2011, transfer_loss: 0.0542, total_Loss: 2.2554, test_loss 2.207802, test_acc: 10.4505
Epoch: [ 9/10], cls_loss: 2.2020, transfer_loss: 0.0542, total_Loss: 2.2562, test_loss 2.207298, test_acc: 10.2218
Epoch: [ 9/10], cls_loss: 2.2020, transfer_loss: 0.0542, total_Loss: 2.2562, test_loss 2.207298, test_acc: 10.2218
Epoch: [10/10], cls_loss: 2.1989, transfer_loss: 0.0542, total_Loss: 2.2532, test_loss 2.206840, test_acc: 10.4734
Epoch: [10/10], cls_loss: 2.1989, transfer_loss: 0.0542, total_Loss: 2.2532, test_loss 2.206840, test_acc: 10.4734
Transfer result: 10.4734

**Comment by jindongwang** [comment]
先试一下，不用Transfer，直接就是普通的深度学习训练，效果是多少呢？

**Comment by Xiao-YiZi** [comment]
> 先试一下，不用Transfer，直接就是普通的深度学习训练，效果是多少呢？

您好，我用sourceonly训练可以达到60%左右，但是没有使用您的代码，所以感觉是数据加载形式的问题？

**Comment by jindongwang** [comment]
那你相应修改一下再试试吧

**Comment by jindongwang** [comment]
@Xiao-YiZi 问题解决了吗？

**Comment by Xiao-YiZi** [comment]
> @Xiao-YiZi 问题解决了吗？

您好，调参后问题解决了，感谢您的回复

**Comment by ZLY980228** [comment]
您好，可以请问具体调什么参数嘛，我也遇到了相同的问题，自己的数据集但是准确率只有个位数

**Comment by Xiao-YiZi** [comment]
> 您好，可以请问具体调什么参数嘛，我也遇到了相同的问题，自己的数据集但是准确率只有个位数

我调整了学习率为0.1以后就可以了你也可以试着把学习率调高点
